src/lib.rs

```diff
@@ -161,14 +182,13 @@ pub struct Sprite3d {
     ///   (though you can go out of bounds without issue)
     pub pivot: Option<Vec2>,
 
-    /// Whether the sprite should support partial alpha.
+    /// Wheather sprite's partial alpha should be handled in a particular way.
     ///
-    /// - `false` (default) only allows fully opaque or fully transparent pixels
+    /// - `None` (default) only allows fully opaque or fully transparent pixels
     ///   (cutoff at `0.5`).
-    /// - `true` allows partially transparent pixels
-    ///   (slightly more expensive, so disabled when not needed).
-    pub partial_alpha: bool,
-
+    /// - `Some(AlphaMode)` allows setting a custom alpha mode. Use `Some(AlphaMode::Blend)`
+    ///   for partially transparent pixels (slightly more expensive).
+    pub alpha_mode: Option<AlphaMode>,
```

Comment:
Could you change this to just be an `AlphaMode` instead of an `Option<AlphaMode>` *(but keep the comments about default behaviour)*? Since sprites will usually be constructed by setting some desired features then calling `..default()` having `None` feels redundant.

Reply:
Absolutely! What if we default to the old behavior with `Mask(0.5)`?